config/routes.rb

```diff
@@ -1,29 +1,10 @@
 Rails.application.routes.draw do
-  root 'auctions#index'
-
-  get '/auctions/winners', to: 'auctions#previous_winners'
-
-  get '/auctions/rules/single-bid', to: 'auctions#single_bid_rules'
-  get '/auctions/rules/multi-bid', to: 'auctions#multi_bid_rules'
-  resources :auctions, only: [:index, :show] do
-    resources :bids, only: [:new, :create, :index] do
-      collection do
-        post :confirm
-      end
-    end
-  end
-
-  resources :bids, only: [:index]
-
-  get '/my-bids', to: 'bids#my_bids'
-
-  resources :users, only: [:update]
-  get 'users/edit' => 'users#edit'
-
+  # Admin needs to go above redirect
```

Comment:
This is left over from my own notes. Because I am currently keeping the RESTful stuff for admin controllers (because I want to wait until we can redo any internal scripts that use them as well), I can't rewrite requests to `/admin/auctions` to `/api/v0/admin/auctions` yet